halo/attest/keeper/common_test.go

```diff
@@ -22,7 +22,9 @@ type mocks struct {
 	namer   *testutil.MockChainNamer
 }
 
-func mockDefaultExpectations(_ sdk.Context, m mocks) {}
+func mockDefaultExpectations(_ sdk.Context, m mocks) {
+	m.namer.EXPECT().ChainName(gomock.Any()).Return("test_chain").AnyTimes()
```

Comment:
since by default we use the default message, maybe we want to make sure that we use the correct chain, e.g:
```suggestion
	m.namer.EXPECT().ChainName(uint64(1)).Return("test_chain").AnyTimes()
```

Reply:
In my upcomming PR I have this:

```go
func TestKeeper_Approve(t *testing.T) {
	t.Skip()
	t.Parallel()
	...
	defaultExpectations := func(_ sdk.Context, m mocks) {
		m.voter.EXPECT().TrimBehind(gomock.Any()).Times(1).Return(0)
	}

	namerCalled := func(times int) expectation {
		return func(_ sdk.Context, m mocks) {
			m.namer.EXPECT().ChainName(uint64(1)).Times(times).Return("test-chain")
		}
	}

```

and then in the table tests:

```go
	expectations: []expectation{
		namerCalled(1),
		defaultExpectations,
	},
```

Note that in `common_test.go` I've defined types for:
```go

type expectation func(sdk.Context, mocks)
type prerequisite func(t *testing.T, k *keeper.Keeper, ctx sdk.Context)

```

I guess we are going to use `namerCalled` often.